haproxy/recipes/ctl.rb

```diff
@@ -18,3 +18,27 @@
 link '/etc/init.d/haproxyctl' do
   to "#{base_path}/haproxyctl/bin/haproxyctl"
 end
+
+### haproxyctl statsd stuff ###
+statsd_host = node['haproxy']['ctl']['statsd']['host']
+statsd_port = node['haproxy']['ctl']['statsd']['port']
+directory '/etc/haproxy/haproxyctl' do
```

Comment:
I would add `recursive true` so the entire path is created. So e.g. if this recipe runs by itself or _before_ haproxy is installed. If `/etc/haproxy` does not exist, it may create a hard failure.